### DISCUSSION TITLE
Pi HDMI Out

### DISCUSSION THREAD

**Discussion by elk-1984** [frntc/RAD#13] [General]
An idle thought and I don't know much at all about the C64 and expanding it, but would it be possible to use the RAD Expansion as an RGB2HDMI like those created for the Spectrum, Oric and Acorn ranges of computers?

**Comment by frntc** [comment]
Afaik the RGB2HDMI works very differently and closest to that might be the Copperdragon for the C64.

In principle it is imaginable to do something similar to the Sidekick20 (or Turbo Chameleon): emulating the VIC2 (and the CPU!) in parallel and outputting the result to HDMI. It's not very high on my list, the limiting factor in general being spare time and many other ideas to try out :-)
